### PR TITLE
es6-modules

### PR DESCRIPTION
**Pull request by easton-morris**

### CODE REVIEW

**Comment by easton-morris** [comment]
do we just screenshot the app?

**Comment by scbowler** [comment]
> do we just screenshot the app?

A screenshot of the working App and your terminal output after running `npm run build`

File: es6-modules/src/components/todo-list.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js:1]: Here you want to import from the `lib/index.js`. We will go over tonight why one is preferred over the other.

```JAVASCRIPT
import { createElement } from '../lib';
```

File: es6-modules/src/components/todo-form.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js:1]: Same here, should be directly from `lib/index.js`

```JAVASCRIPT
import { createElement } from '../lib';
```

File: es6-modules/src/components/todo-app.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { createElement, noop } from '../lib';
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js:1]: This is the benefit of importing from the `lib/index.js`. You can import multiple things from the `lib` folder on 1 line.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but a couple updates needed. Let me know if you have any questions.

**Comment by easton-morris** [comment]
image

**Comment by easton-morris** [comment]
image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**